packages/trpc/server/document-router/router.ts

```diff
@@ -1,5 +1,6 @@
 import { TRPCError } from '@trpc/server';
```

Comment:
This implementation won't work. Any changes to the PDF after signing will invalidate it.

We should collect whether they want the audit log attached to the pdf as a document option, with a team preference.

Then the dropdown will let you optionally download the additional items which will be done as separate downloads.

e.g. `filename_signed.pdf`, `filename_audit.pdf`, `filename_certificate.pdf`